packages/webcomponents/src/components/checkout-wrapper/checkout-wrapper.tsx

```diff
@@ -24,43 +29,80 @@ export class CheckoutWrapper {
     checkoutStore.accountId = this.accountId;
   }
 
+  connectedCallback() {
+    this.observer = new MutationObserver(() => {
+      this.queryFormRefs();
+    });
+
+    this.observer.observe(this.hostEl, {
+      childList: true,
+      subtree: true
+    });
+  }
+
   componentDidLoad() {
+    this.queryFormRefs();
+  }
+
+  private queryFormRefs() {
     this.cardFormRef = this.hostEl.querySelector('justifi-card-form');
+    this.bankAccountFormRef = this.hostEl.querySelector('justifi-bank-account-form');
   }
 
   @Method()
   async validate(): Promise<boolean> {
-    if (this.cardFormRef) {
-      return this.cardFormRef.validate();
+    if (this.cardFormRef || this.bankAccountFormRef) {
+      const formToValidate = this.cardFormRef || this.bankAccountFormRef;
+      return formToValidate.validate();
     } else {
       throw new Error('CardForm component not found');
     }
   }
 
   @Method()
-  async tokenizePaymentMethod({ addressPostalCode }: { addressPostalCode?: number }): Promise<any> {
-    if (this.cardFormRef) {
-      const isValid = await this.cardFormRef.validate();
+  async tokenizePaymentMethod(billingInfo: BillingInfo): Promise<any> {
+    if (this.cardFormRef || this.bankAccountFormRef) {
+      const formToTokenize = this.cardFormRef || this.bankAccountFormRef;
+      const isValid = await formToTokenize.validate();
       if (!isValid) {
         throw new Error('Card form is not valid');
       }
-      return this.cardFormRef.tokenize({
+      return formToTokenize.tokenize({
         clientId: this.authToken,
         paymentMethodMetadata: {
           accountId: this.accountId,
-          address_postal_code: addressPostalCode
+          ...billingInfo
         },
         account: this.accountId,
       });
     } else {
-      throw new Error('CardForm component not found');
+      this.errorEvent.emit({
+        message: 'CardForm component not found',
```

Comment:
Should this message be updated? Maybe use the tag names instead of the class names? Something like `Could not find justifi-card-form or justifi-bank-account-form`?